doc/versionHistory.md

```diff
@@ -1,5 +1,9 @@
 # Version History
 
+0.1.8
+
+- Added main m2cellcpp program.
```

Comment:
Maybe "Added the main executable `m2cellcpp`." to avoid the confusion (people might think the word of main here is the executable).